go/vt/topo/helpers/copy.go

```diff
@@ -165,7 +165,7 @@ func CopyShardReplications(ctx context.Context, fromTS, toTS *topo.Server) {
 			for _, cell := range cells {
 				sri, err := fromTS.GetShardReplication(ctx, cell, keyspace, shard)
 				if err != nil {
-					log.Fatalf("GetShardReplication(%v, %v, %v): %v", cell, keyspace, shard, err)
+					log.Errorf("GetShardReplication(%v, %v, %v): %v", cell, keyspace, shard, err)
```

Comment:
I don't understand this change. Seems to be unrelated, right?

Reply:
`Fatalf` means the func will exit, so the `continue` is unreachable. If this is really a fatal error, then we could alternatively delete the next line which says `continue`.

Reply:
This is not test-code. I don't think it's safe to change this behaviour here.

Feel free to remove the `continue` after though - like you correctly point out, `Fatal` will exit the method anyway.

Reply:
DONE